src/components/Help/HelpCenter/index.tsx

```diff
@@ -0,0 +1,311 @@
+            <ul>
+              <li>
+                <Link to={{ description: { other: ['about', 'interpro'] } }}>
+                  InterPro
+                </Link>
+              </li>
+              <li>
+                <Link to={{ description: { other: ['about', 'consortium'] } }}>
+                  InterPro consortium
+                </Link>
+              </li>
+              <li>
+                <Link to={{ description: { other: ['about', 'team'] } }}>
+                  Team
+                </Link>
+              </li>
+            </ul>
+          </li>
+        </div>
+        <div className={css('flex-box')}>
+          <span
+            className={css('icon', 'icon-common', 'xl')}
+            data-icon="&#xf10d;"
+          />{' '}
+          <h5>How to cite us</h5>
+          <ul>
+            <li>
+              <Link
+                href={`${config.root.readthedocs.href}citing.html`}
+                className={css('ext')}
+                target="_blank"
+              >
+                Citing InterPro
+              </Link>
+            </li>
+            <li>
+              <Link
+                href="https://pfam-docs.readthedocs.io/en/latest/citing-pfam.html"
+                className={css('ext')}
+                target="_blank"
+              >
+                Citing Pfam
+              </Link>
+            </li>
+          </ul>
+        </div>
+        <div className={css('flex-box')}>
+          <span
+            className={css('icon', 'icon-common', 'xl')}
+            data-icon="&#xf2fd;"
+          />{' '}
+          <h5>InterProScan</h5>
+          <ul>
+            <li>
+              <Link
+                href="https://interproscan-docs.readthedocs.io/"
+                className={css('ext')}
+                target="_blank"
+              >
+                InterProScan documentation
+              </Link>
+            </li>
+            <li>
+              <Link
+                href="//www.ebi.ac.uk/jdispatcher/docs/"
+                className={css('ext')}
+                target="_blank"
+              >
+                Job Dispatcher documentation
+              </Link>
+            </li>
+          </ul>
+        </div>
+
+        <div className={css('flex-box')}>
+          <span
+            className={css('icon', 'icon-common', 'xl')}
+            data-icon="&#xf085;"
+          />
+          <h5>Technical corner</h5>
+          <ul>
+            <li>
+              <Link
+                href="          https://github.com/ProteinsWebTeam/interpro7-api/tree/master/docs"
+                className={css('ext')}
+                target="_blank"
+              >
+                Programmatic access
+              </Link>
+            </li>
+            <li>
+              <Link
+                href="https://www.ebi.ac.uk/interpro/api/static_files/swagger/"
+                className={css('ext')}
+                target="_blank"
+              >
+                Open API specification
```

Comment:
```suggestion
                Swagger UI
```

(My mistake, I wrote "OpenAPI specification" in the Jira ticket)